lib/checks/canDistribute.js

```diff
@@ -0,0 +1,24 @@
+const Node = require('../node');
+
+function canDistribute(node) {
+  // For the special case when the node is in the form (x - 2)^2
+  // First check if is power node and the exponent is a constant
+  // Then check if the base is a polynomial
+
+  const checks = Node.Type.isOperator(node)
+        && node.op === '^'
+        && Node.Type.isParenthesis(node.args[0])
```

Comment:
Do we have to check that the thing inside the parenthesis is the sum or difference of exactly two terms?